pages/documentation/grid/model/extensions.md

```diff
@@ -262,12 +262,12 @@ A load is described by its active power setpoint $$P0$$ and its reactive power s
 - In the total amount of active power what is fixed and what is time-dependant (also called variable). The time-dependant part can be adjusted for production equals consumption.
 - In the total amount of reactive power what is fixed and what is time-dependant (also called variable).
 
-| Attribute | Type | Unit | Required | Default value | Description |
-| --------- | ---- | ---- | -------- | ------------- | ----------- |
```

Comment:
@annetill, here ware the units wrong. The rest is just table formatting reported by sonarlint.